openfe/setup/methods/openmm/equil_rbfe_methods.py

```diff
@@ -0,0 +1,533 @@
+        arbitrary_types_allowed = True
+
+    minimization_steps = 10000
+    equilibration_length: unit.Quantity
+    production_length: unit.Quantity
+
+    # reporter settings
+    output_filename = 'rbfe.nc'
+    checkpoint_interval = 50 * unit.timestep
+    checkpoint_storage = Union[str, None]
+
+    @validator('equilibration_length', 'production_length')
+    def is_time(cls, v):
+        # these are time units, not simulation steps
+        if not v.is_compatible_with(unit.picosecond):
+            raise ValueError("Durations must be in time units")
+        return v
+
+    @validator('minimization_steps', 'equilibration_length',
+               'production_length', 'checkpoint_interval')
+    def must_be_positive(cls, v):
+        if v <= 0:
+            errmsg = ("Minimization steps, MD lengths, and checkpoint "
+                      "intervals must be positive")
+            raise ValueError(errmsg)
+        return v
+
+
+class RelativeLigandTransformSettings(BaseModel):
+    class Config:
+        arbitrary_types_allowed = True
+
+    # Things for creating the systems
+    system_settings: SystemSettings
+    topology_settings: TopologySettings
+
+    # Alchemical settings
+    alchemical_settings: AlchemicalSettings
+
+    # MD Engine things
+    engine_settings = OpenMMEngineSettings()
+
+    # Sampling State defining things
+    integrator_settings: IntegratorSettings
+    barostat_settings: BarostatSettings
+    sampler_settings: SamplerSettings
+
+    # Simulation run settings
+    simulation_settings: SimulationSettings
+
+    # solvent model?
+    solvent_padding = 1.2 * unit.nanometer
+
+
+class RelativeLigandTransformResults:
+    """Dict-like container for the output of a RelativeLigandTransform"""
+    def __init__(self, settings: RelativeLigandTransformSettings):
+        self._parent_settings = settings
+        fn = self._parent_settings.simulation_settings.output_filename
+        self._reporter = multistate.MultiStateReporter(fn)
+        self._analyzer = multistate.MultiStateSamplerAnalyzer(self._reporter)
+
+    def dG(self):
+        """Free energy difference of this transformation
+
+        Returns
+        -------
+        dG : unit.Quantity
+          The free energy difference between the first and last states. This is
+          a Quantity defined with units.
+
+        TODO
+        ----
+        * Check this holds up completely for SAMS.
+        """
+        dG, _ = self._analyzer.get_free_energy()
+        dG = (dG[0, -1] * self._analyzer.kT).in_units_of(
+            unit.kilocalories_per_mol)
+
+        return dG
+
+    def dG_error(self):
+        """The uncertainty/error in the dG value"""
+        _, error = self._analyzer.get_free_energy()
+        error = (error[0, -1] * self._analyzer.kT).in_units_of(
+            unit.kilocalories_per_mol)
+
+        return error
+
+
+class RelativeLigandTransform(FEMethod):
+    """Calculates the free energy of an alchemical ligand swap in solvent
```

Comment:
```suggestion
    """Calculates the relative free energy of an alchemical ligand transformation.
```

Technically the way this is set it up, it could either be a complex, solvent, or gas phase transformation?